### ISSUE TITLE
Perk for Holding Steem Power: Site Avatar/Profile Picture

### ISSUE THREAD

**Issue by Demotruk** [steemit/steem#354]
There is a very large gap between the minimum amount of Steem that an account typically begins with (~$10), and a large amount which begins to have meaningful value to the user in terms of voting power, curation rewards etc. (~$3000) There's virtually no value in being a large minnow vs a small minnow. You have some protection from smaller trolls, and it can be a speculative investment, but that will not be enough to drive millions of people to value Steem Power in the amounts that they could afford to buy, without being a significant investment for them.

I wrote a related post on SteemIt about this [here](https://steemit.com/steem-ideas/@demotruk/website-perks-for-steem-power) and it was suggested to post it in the GitHub project.

In short I think we need perks on the website for those who hold Steem Power which kick in at relatively small amounts. Taking inspiration from Gaia Online as well as countless online video games, SteemIt.com could offer a custom avatar to all users which have a certain minimum stake in the project. $20 worth perhaps, although it may need to be denominated in Steem Power so that nobody loses a perk simply because the price went down. There are loads of other perks which could be applied at various thresholds, you could take inspiration from Reddit Gold or Reddit Enhancement Suite features.

**Comment by Demotruk** [comment]
I see I posted this in the wrong section of the site. Is it possible to have it moved?
